src/main/java/dev/blaauwendraad/masker/json/KeyContainsMasker.java

```diff
@@ -316,17 +273,17 @@ private void maskNumber(MaskingState maskingState, KeyMaskingConfig keyMaskingCo
      */
     private void maskBoolean(MaskingState maskingState, KeyMaskingConfig keyMaskingConfig) {
         int targetValueLength = AsciiCharacter.isLowercaseT(maskingState.byteAtCurrentIndex()) ? 4 : 5;
```

Comment:
I will add a comment that this is 4 because of `true` or 5 because of either `null` or `false`

Reply:
Here it's only true (4) or false (5). Did you mean visitValue method where we do `'t', 'n' -> 4`?

Reply:
Ah, you are right